pages/Product/subscriptions.js

```diff
@@ -75,6 +75,7 @@ export default function product(subscribe) {
       .filter(({ action }) => (
         action.type === RECEIVE_PRODUCT || action.type === RECEIVE_PRODUCT_CACHED
       ))
+      .filter(({ getState }) => getCurrentBaseProductId(getState()) !== null)
```

Comment:
I would rather put a check to the `shouldFetchReviews$` subscription, or only emit when the pdp or one of their subpages is active.

Reply:
i added a check to the getProductReviews action and would like to keep a check here (url== 'item'), because for me it's wrong that the shouldFetchReviews$ streams emits everytime someone dispatchs RECEIVE_PRODUCT.